src/app/new-page/new-page.component.scss

```diff
@@ -0,0 +1,13 @@
+.table th, .table td {
+    padding: .25rem;
+    font-size: .875rem;
+}
+
+@media (min-width: 572px) { 
```

Comment:
This could be replaced with bootstrap media queries mixins - [](https://www.dev-tips-and-tricks.com/use-bootstrap-4-media-query-mixins)

Reply:
https://www.dev-tips-and-tricks.com/use-bootstrap-4-media-query-mixins